pytest_rts/utils/test_selection.py

```diff
@@ -14,9 +16,7 @@
     get_current_head_hash,
 )
 
-# https://github.com/PyCQA/pylint/issues/3876
-# pylint: disable=inherit-non-class
-# pylint: disable=too-few-public-methods
+
 class UpdateData(NamedTuple):
```

Comment:
Wait, I've missed it previously, sorry! -- why do you inherit from NamedTuple? This is wrong. What you should do is
```
UpdateData = NamedTuple("UpdateData", [
    ("changed_lines_test", Dict[str, List[int]]),
    ...])
```

Reply:
Changed to be like this